<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Scenario: creation is successful with valid username and password" tests="4" skipped="0" failures="2" errors="0" timestamp="2017-11-19T21:31:49" hostname="fuxi16" time="0.022">
  <properties/>
  <testcase name="Given command new user is selected" classname="Scenario: creation is successful with valid username and password" time="0.0"/>
  <testcase name="When username &quot;pekka&quot; and password &quot;akkep!123&quot; are entered" classname="Scenario: creation is successful with valid username and password" time="0.01"/>
  <testcase name="Then system will respond with &quot;new user registered&quot;" classname="Scenario: creation is successful with valid username and password" time="0.007">
    <failure message="java.lang.AssertionError" type="java.lang.AssertionError">java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at ohtu.Stepdefs.system_will_respond_with(Stepdefs.java:38)
	at ✽.Then system will respond with &quot;new user registered&quot;(ohtu/new_user.feature:6)
</failure>
  </testcase>
  <testcase name="classMethod" classname="Scenario: creation is successful with valid username and password" time="0.023">
    <failure message="java.lang.AssertionError" type="java.lang.AssertionError">java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at ohtu.Stepdefs.system_will_respond_with(Stepdefs.java:38)
	at ✽.Then system will respond with &quot;new user registered&quot;(ohtu/new_user.feature:6)
</failure>
  </testcase>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
